web/themes/custom/cloudy/src/components/badge/badge.twig

```diff
@@ -0,0 +1,11 @@
+{% set isPill = isPill|default(false) %}
+{% set isNavAlert = isNavAlert|default(false) %}
+
+{% set classes = [
+  'cloudy-badge',
+  'cloudy-badge--' ~ type,
+  isPill ? 'cloudy-badge-pill',
```

Comment:
lets treat pill as a modifier and use `cloudy-badge--pill`, you will need to update the scss as well.